packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_service_metrics_aggregator.ts

```diff
@@ -45,6 +45,9 @@ export function createServiceMetricsAggregator(flushInterval: string) {
             sum: 0,
             value_count: 0,
           },
+          'data_stream.type': 'metrics',
```

Comment:
Any specific reason to add these 3 values to all the APM aggregators ?

Concern i see is these 3 values together represent the actual DataStream Name where documents get indexed. And we have the data stream name logic defined in the routing transform here which is different.
`packages/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client/get_routing_transform.ts`